app/lib/analyzer/model_properties.dart

```diff
@@ -47,8 +47,8 @@ class AnalysisStatusProperty extends StringProperty {
           return AnalysisStatus.failure;
         case 'success':
           return AnalysisStatus.success;
-        case 'deprecated':
```

Comment:
It won't be a problem in prod, but do we have entries with status=='deprecated' stored in dev?

Reply:
We have, but it is unlikely to cause problems.

Reply:
It'll cause an exception to be thrown here, right?

Reply:
It will. But it will happen only if the most-recent analysis gets deleted, and only the older `'deprecated'` Analysis instance is available. The automatic removal of old Analysis instances will remove that soon and it won't cause issues.

Reply:
Right. No worries, then. :)